packages/bigtable/src/table.js

```diff
@@ -807,8 +808,8 @@ Table.prototype.insert = function(entries, callback) {
  * table.mutate(entries, callback);
  *
  * //-
- * // A stream mode is also provided, this is useful when making a large number
- * // of mutations.
+ * // An event emitter mode is also provided, this is useful when making a large
+ * // number of mutations.
```

Comment:
If you don't provide a callback, an EventEmitter is returned. Listen for the `error` event to catch API and mutation errors, and `complete` for when the API request has completed.